.github/workflows/release.yml

```diff
@@ -10,6 +10,11 @@ jobs:
     environment: release
     container:
       image: ghcr.io/cockpit-project/release
+    permissions:
+      # create GitHub release
+      contents: write
+      # pul the release container
```

Comment:
pul!

For what it's worth, it also works fine without this, since it's public.

Reply:
D'oh, thanks! I dropped this bit and adjusted the commit message.